rpc-client/src/nonblocking/rpc_client.rs

```diff
@@ -7,6 +7,7 @@
 //! [JSON-RPC]: https://www.jsonrpc.org/specification
 
 pub use crate::mock_sender::Mocks;
+use crate::mock_sender::MocksMap;
```

Comment:
CI is complaining about this -- be sure to `./cargo nightly fmt --all` from the root of the repo and commit the result